docs/core/additional-tools/wcf-web-service-reference-guide.md

```diff
@@ -0,0 +1,78 @@
+﻿--- 
```

Comment:
remove the hidden character here that is causing the metadata section not to render as a table on preview